Source/ProjectRimFactory/Common/PRFGameComponent.cs

```diff
@@ -17,19 +17,20 @@ public class PRFGameComponent : GameComponent {
             base.ExposeData();
             Scribe_Collections.Look(ref specialScupltures, "specialSculptures");
         }
-        public static void RegisterAssemblerQueue(IAssemblerQueue queue)
+        public void RegisterAssemblerQueue(IAssemblerQueue queue)
         {
             //enshure that there are no duplicates
             //im shure there is a better way
-            AssemblerQueue.RemoveAll(q => queue.ToString() == q.ToString());
+           // AssemblerQueue.RemoveAll(q => queue.ToString() == q.ToString());
 
             AssemblerQueue.Add(queue);
+#if DEBUG
```

Comment:
C# has this fancy thing where any call to `Debug.Message` will *not be compiled* unless it's a debug build! ( See `[System.Diagnostics.Conditional("DEBUG")]` in Common/Debug.cs)  So the #if DEBUG is not needed (unless you need to do any pre-calculation for the Debug.Message() call)